roles/load_aware_undeploy_trimaran/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+- name: Ensure Trimaran is deployed
+  command: oc get ns trimaran --ignore-not-found -oname
+  register: has_trimaran
+
+- name: Delete Trimaran resources
+  command: 
+    oc delete -f {{ trimaran_setup_config }} --ignore-not-found
+  when: has_trimaran.stdout | length > 0
+
+- name: Ensure user applications monitor config is deployed
+  command: oc get configmap cluster-monitoring-config --ignore-not-found -oname -n openshift-monitoring
+  register: has_user_monitoring
+
+- name: Delete user monitoring configmap
+  command:
+    oc delete -f {{ user_applications_monitor_config }} --ignore-not-found
+  when: has_user_monitoring | length > 0
```

Comment:
this should go away, as discussed in the other thread

just FYI, `when: has_user_monitoring | length > 0` this isn't necessary, as `--ignore-not-found` prevents `oc delete` from crashing (complaining) if the ConfigMap does not exist